README.md

```diff
@@ -149,13 +136,36 @@ md5sum: WARNING: 1 computed checksum did NOT match
 
 This means that the built ROM isn't the same as the base one, so something went wrong or some part of the code doesn't match.
 
+### Cygwin
+
+If you want to use Cygwin, you will need to:
+
+* Download and install [Git Bash](https://git-scm.com/download/win).
+* Download and install [Cygwin](https://cygwin.com).
+* [Build and install mips-linux-binutils](docs/BUILDING_BINUTILS_CYGWIN.md).
+
+Once mips-linux-binutils is installed you will need to install the following packages using Cygwin's installer:
+
+* libiconv
+* dos2unix
+* python3
+
+Then you can continue from step 3 of the Linux instructions.
+
+Note that, before building anything, you will need to run the following commands to fix line endings:
+
+```bash
+dos2unix fixle.sh
+./fixle.sh
+```
+
 ### Docker
```

Comment:
I think there was a discussion to remove docker because no one was using or docker and it would quickly become outdated due to lack of support for it, so this entire section could just be removed?

Reply:
we ended up deciding to keep it around at least while it still works. but if it becomes a pain to maintain then it can be removed later